### DISCUSSION TITLE
ESP32 builds are not using Tasmota board definitions

### DISCUSSION THREAD

**Discussion by Jason2866** [benzino77/tasmocompiler#151] [General]
Tasmota uses optimized / corrected board definitions for Odroid-go and M5 Stack core2.
For M5 Stack core2 is a complete wrong board definition used.
The Odroid-go and M5 Stack core 2 board definition from Platformio is not complete/faulty
I recommend to use the Tasmota ones. Without unexpected behaviour will probably occur.

Greets Jason0815

@benzino77 Great work!

**Comment by benzino77** [comment]
For Odroid-Go the `board` entry used by TasmoCompiler is the same as Tasmota: `odroid_esp32`
([TasmoCompiler](https://github.com/benzino77/tasmocompiler/blob/master/src/components/AppStepper/FeaturesStep/AvailableBoards.js#L93) and [Tasmota](https://github.com/arendst/Tasmota/blob/development/platformio_tasmota_env32.ini#L48))

For M5 Stack Core2 there is a difference. Tasmota has set it up for `odroid_esp32` and TasmoCompiler is using `m5stack-grey` which is ... not correct, as you suggested - have to fix it.

@Jason2866 - thanks for pointing this out!
